src/Model/Actions.re

```diff
@@ -128,6 +128,8 @@ type t =
   | StatusBarDisposeItem(int)
   | StatusBar(StatusBarModel.action)
   | ViewCloseEditor(int)
+  | ViewNextEditor
+  | ViewPreviousEditor
```

Comment:
OK that makes sense. It will annoy me not having type-checking on the string, and copy and pasting it everywhere, but I can change it until you add `CommandTriggered`.